src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -234,6 +238,13 @@ class GroupDetails extends React.Component<Props, State> {
   }
 
   renderError() {
+    const {organization, location} = this.props;
+    const projects = organization.projects;
+    const projectId = location.query.project;
+
+    const project = projects.filter(proj => proj.id === projectId);
+    const projectSlug = project.find(proj => proj.id === projectId)?.slug;
```

Comment:
pretty sure this can just be one line

```suggestion
    const projectSlug = projects.find(proj => proj.id === projectId)?.slug;
```

Reply:
woops 👍